addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -38,9 +38,11 @@
                             <i class="fa fa-lg fa-phone text-700"/>
                         </button>
                     </t>
-                    <button class="btn" t-ref="addUsers" title="Add Users" data-action="add-users" t-on-click.stop="toggleInviteForm">
-                        <i class="fa fa-lg fa-user-plus text-700"/>
-                    </button>
+                    <t t-if="thread.type !== 'mailbox'">
```

Comment:
missing test, and we often put the t-if directly on the main tag now

otherwise ok

Reply:
tests added there: https://github.com/odoo-dev/odoo/pull/1683